src/sentry/templates/sentry/auth_close.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <script>
+  // sends message to window that opened this popup if user is authenticated and closes the window
+  function extractOrigin() {
```

Comment:
Can we fix the indentation in this `<script>` block? `checkAuthChange` is indented an extra level.